packages/RichTextEditing-Tests.package/RichTextStructureEditorStructureTests.class/instance/hasNoStructureForText..st

```diff
@@ -3,5 +3,5 @@ hasNoStructureForText: aText
 
 	^ (aText runs copyFrom: 0 to: aText size)
 		allSatisfy: [:attributes | attributes
-			noneSatisfy: [:attribute | attribute isTextStructure]]
+			noneSatisfy: [:attribute | (attribute isTextStructure) and: ( attribute isNullObject not)]]
```

Comment:
I think using and with [] is better to avoid evaluating the statement if the first already failed
```suggestion
			noneSatisfy: [:attribute | (attribute isTextStructure) and: [attribute isNullObject not]]]
```